Base/usr/share/man/man1/ntpquery.md

```diff
@@ -1,21 +1,28 @@
 ## Name
 
-ntpquery
+ntpquery - Query the time from a NTP server
 
 ## Synopsis
 
-```sh
-$ ntpquery [--adjust] [--set] [--verbose] [host]
+```**sh
+$ ntpquery [-v] [-a|-s] [host]
 ```
 
-## Options:
+## Description
+
+`ntpquery` contacts a NTP server (by default time.google.com), and optionally sets or
+adjusts (i.e. slow down or speed up) the system time, as long as the command is run as root.
```

Comment:
Perhaps the mention that this needs to be run as root is unnecessary. At least, I don't see this same type of comment having been added to the other manpages that required a certain privilege.

Reply:
`groupadd(8)` does mention that it needs to be run as root.